To-do list/Controllers/TaskDetailsViewController.swift

```diff
@@ -7,91 +7,48 @@
-        if choosenTableViewCell {
-            titleTextField.text = users[userIndex].task?[taskIndex].title
-            descriptionTextField.text = users[userIndex].task?[taskIndex].description
-            
-            switch users[userIndex].task?[taskIndex].status {
-            case "in progress":
-                statusButtonOutlet.backgroundColor = .systemOrange
-                statusButtonOutlet.setTitle("in progress", for: .normal)
-            case "done":
-                statusButtonOutlet.backgroundColor = .systemRed
-                statusButtonOutlet.setTitle("done", for: .normal)
-            case "to do":
-                statusButtonOutlet.backgroundColor = .systemGreen
-                statusButtonOutlet.setTitle("to do", for: .normal)
-            default:
-                break
-            }
-        }
+        prepareUIElements()
     }
     
-    @IBAction func statusButton(_ sender: UIButton) {
-        if choosenTableViewCell == false {
-            counter += 1
-            switch counter {
-            case 1:
-                sender.backgroundColor = .systemOrange
-                sender.setTitle("in progress", for: .normal)
-                status = sender.currentTitle ?? ""
-            case 2:
-                sender.backgroundColor = .systemRed
-                sender.setTitle("done", for: .normal)
-                status = sender.currentTitle ?? ""
-            case 3:
-                sender.backgroundColor = .systemGreen
-                sender.setTitle("to do", for: .normal)
-                status = sender.currentTitle ?? ""
-                counter = 0
-            default:
-                break
-            }
-        } else {
-            var currentStatus = users[userIndex].task?[taskIndex].status
-            
-            switch currentStatus {
-            case "in progress":
-                currentStatus = "done"
-                sender.backgroundColor = .systemRed
-                sender.setTitle("done", for: .normal)
-                users[userIndex].task?[taskIndex].status = currentStatus!
-                UserDefaultsManager.shared.setValueForUser(value: users)
-            case "done":
-                currentStatus = "to do"
-                sender.backgroundColor = .systemGreen
-                sender.setTitle("to do", for: .normal)
-                users[userIndex].task?[taskIndex].status = currentStatus!
-                UserDefaultsManager.shared.setValueForUser(value: users)
-            case "to do":
-                currentStatus = "in progress"
-                sender.backgroundColor = .systemOrange
-                sender.setTitle("in progress", for: .normal)
-                users[userIndex].task?[taskIndex].status = currentStatus!
-                UserDefaultsManager.shared.setValueForUser(value: users)
-            default:
-                break
-            }
+    func prepareUIElements() {
+            titleTextField.text = task.title
```

Comment:
remove extra space indents. (4 spaces indent is correct one)

Reply:
done